src/assets/custom/js/websiteNavigation.test.js

```diff
@@ -13,12 +13,18 @@ const OPEN_SUB_MENU_CLASS = "website-navigation-sub-menu--open";
 const HEADER_HAS_OPEN_SUBMENU_CLASS = "website-header--has-open-submenu";
 
 const fakeSubMenuScrollHeight = 999;
+Object.defineProperty(HTMLElement.prototype, "scrollHeight", {
+  configurable: true,
+  value: fakeSubMenuScrollHeight,
+});
 
 let header;
 let menu;
 let menuToggle;
-let subMenu;
-let subMenuToggle;
+let subMenu_1;
```

Comment:
Bit picky... but is there a better naming for this, than using 1 and 2 ?

Reply:
Renamed these in this commit :https://github.com/codurance/site/pull/1931/commits/ee65f103d9f5319ed2b3f4bd70e530d2fced2a4c